src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -741,6 +743,51 @@ def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
     # Check Release Existance
     release_namespace = get_release_namespace(kube_config, kube_context, helm_client_location)
 
+    # Check forced delete flag
+    if(force_delete):
+
+        confirmation_message = "You can check using 'kubectl config get-contexts' to check if your current context is pointing to the the right cluster. \n" + "Are you sure you want to execute the delete command:"
+        utils.user_confirmation(confirmation_message, False)
```

Comment:
Please add a new parameter `--yes/-y` to allow the customer to skip the step of prompting confirmation. Otherwise, when users use the `--force` parameter in the automation scripts, the step of prompting confirmation will block the execution of the scripts

Reply:
```suggestion
        if not yes:
               confirmation_message = "You can check using 'kubectl config get-contexts' to check if your current context is pointing to the the right cluster. \n" + "Are you sure you want to execute the delete command:"
               utils.user_confirmation(confirmation_message, False)
```

Reply:
https://github.com/Azure/azure-cli-extensions/blob/048d2ba8a367a8956aad146b0575ceb87c8a94a9/src/connectedk8s/azext_connectedk8s/commands.py#L27

In addition, you need to remove the declaration of `confirmation=True` in the commands.py, and then implement the prompting confirmation in the validator or custom method.